src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -978,7 +982,9 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
             }
         }
 
-        internal sealed override DeclarationScope Scope => _packedFlags.Scope;
+        internal sealed override DeclarationScope DeclaredScope => _packedFlags.Scope;
```

Comment:
Is `PEParameterSymbol.DeclaredScope` reachable (aside from the use on next line)?

Reply:
It looks like `PEParameterSymbol.DeclaredScope` is only used below, so perhaps we could limit `DeclaredScope` to source symbols only. That change may be a little involved though, so perhaps that could be done later if necessary.

Reply:
[https://github.com/dotnet/roslyn/issues/63078](https://github.com/dotnet/roslyn/issues/63078)